src/libs/actions/Policy/Policy.ts

```diff
@@ -348,12 +349,22 @@ function hasActiveChatEnabledPolicies(policies: Array<OnyxEntry<PolicySelector>>
 /**
  * Delete the workspace
  */
-function deleteWorkspace(policyID: string, policyName: string, lastAccessedWorkspacePolicyID: string | undefined, lastUsedPaymentMethods?: LastPaymentMethod) {
+function deleteWorkspace(
+    policyID: string,
+    policyName: string,
+    lastAccessedWorkspacePolicyID: string | undefined,
+    policyCardFeeds: CardFeeds | undefined,
```

Comment:
Just fixed the conflict. @ishpaul777 do you think we should just get the card feeds from Onyx.connectWithoutView since the data doesn't affect rendering?